### DISCUSSION TITLE
Questions about "elements" and "type_map"

### DISCUSSION THREAD

**Discussion by manbujin** [deepmodeling/dpgen#1194] [Q&A] [answered]
Hello！
When I trained the quinternary potential function, the initial data I prepared included both the quinternary and binary models.
When preparing the binary data, my elements and type_map are written as ["Ti","C"].
I would like to ask if this will have an effect on the element mapping when I train the quinternary potential function？Or should I change the type_map to ["W", "Mo", "Ta", "Zr", "Ti", "C"] and elements to ["Ti", "C"]？
Thank you so much！

**Comment by wanghan-iapcm** [comment, marked as answer]
The binary data may be kept as is if the `type_map.raw`in the binary data folders are correctly provided

**Reply by manbujin** [reply]
Thank you very much for your answer.
